openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -244,22 +246,24 @@ def load_casting(project_name, shot_name) -> Set[OpenpypeContainer]:
 
     # Load downloaded subsets
     containers = []
+    all_datablocks = set()
     for representation in representations:
         try:
-            container, _datablocks = load_subset(
+            container, datablocks = load_subset(
                 project_name,
                 representation,
-                f"Link{representation['context']['family'].capitalize()}Loader",
+                f"Link",
```

Comment:
Are we sure we never use `LinkWorkfileLoader` ?

Reply:
They have fixed it on my demand, it is only available on `workfile` family, then yes, we are safe.

https://github.com/Tilix4/OpenPype/blob/re_blender_build_workfile_seamless/openpype/hosts/blender/plugins/load/import_workfile.py